infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServerCmdLine.java

```diff
@@ -29,6 +29,7 @@ private CorfuServerCmdLine() {
                     + "\tcorfu_server (-l <path>|-m) [-nsNA] [-a <address>|-q <interface-name>] "
                     + "[--network-interface-version=<interface-version>] "
                     + "[--max-replication-data-message-size=<msg-size>] "
+                    + "[--max-replication-write-size=<max-replication-write-size>] "
```

Comment:
max-replication-data-message-size also seems similar to max-replication-write-size ?

Reply:
its slightly different...
max-replication-data-message-size is for the size of LR data that the SOURCE will generate
and max-replication-write-size is the size of data written in a single transaction on the SINK.

Reply:
Thanks. It is better to write this in the comment section :)